go.mod

```diff
@@ -13,7 +13,7 @@ require (
 	github.com/containernetworking/cni v1.1.2
 	github.com/containernetworking/plugins v1.4.1
 	github.com/emicklei/go-restful/v3 v3.12.0
-	github.com/envoyproxy/go-control-plane v0.12.0
+	github.com/envoyproxy/go-control-plane v0.12.1-0.20240419124334-0cebb2f428b3
```

Comment:
@lobkovilya can we do it? I think replace won't work

Reply:
yes the replace won't work in this case, I checked locally